src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -1190,3 +1207,196 @@ def run(self):
                 self._future._task_state = FINISHED
 
         return da_rois
+
+def get_tiled_areas(
+    pos: Dict[str, float],
+    streams: List[Stream],
+    tiles_nx: int,
+    tiles_ny: int,
+    overlap: float,
+    tiling_rng: Dict[str, list],  # TODO: make optional, use axes range otherwise
+    selected_grids: List[str] = None,
+    sample_centers: Dict[str, Tuple[float]] = None,
+    whole_grid: bool = False,
+    rel_bbox: tuple = SAMPLE_USABLE_BBOX_TEM_GRID,
+) -> List[Tuple[float]]:
+    """
+    Compute the bounding boxes of all areas to acquire.
+    :param pos: the current position of the stage
+    :param streams: the streams to acquire
+    :param tiles_nx: the number of tiles in the x direction
+    :param tiles_ny: the number of tiles in the y direction
+    :param overlap: the overlap between tiles (in percentage)
+    :param tiling_rng: the tiling range along x and y axes as (xmin, ymin, xmax, ymax), or (xmin, ymax, xmax, ymin)
+    :param selected_grids: the grids to acquire
+    :param sample_centers: the centers of the sample grids
+    :param whole_grid: if True, the whole grid is acquired
+    :param rel_bbox: the relative bounding box of the sample grid (xmin, ymin, xmax, ymax)
+    return: the bounding boxes (xmin, ymin, xmax, ymax in physical coordinates)
+    of all areas to acquire.
+    """
+    if not whole_grid:
+        area = compute_area_size(
+            pos=pos,
+            streams=streams,
+            tiles_nx=tiles_nx,
+            tiles_ny=tiles_ny,
+            tiling_rng=tiling_rng,
+            overlap=overlap,
+        )
+        if area is None:
+            return []
+        else:
+            return [area]
+
+    # TODO: for now this is only for the MIMAS, but eventually, on the METEOR,
```

Comment:
Is this still a `TODO`?

Reply:
Yes, the sample centers still aren't implemented correctly for meteor